### PR TITLE
update annotation service connection

### PR DESCRIPTION
**Pull request by birdage**
<b>do not merge yet</b>

@DanielJMaher FYI

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@FBRTMaka Can you review??  maybe pull down and test?

**Comment by FBRTMaka** [comment]
@DanielJMaher , @birdage sure! I'll have a look now

File: ooiservices/app/main/annotation.py [hunk 1]
```diff
@@ -23,23 +23,27 @@
...
-            url = "/".join([UFRAME_DATA,mooring, platform, instrument, stream_type, stream + query])
+        return jsonify( {'annotations' : data }), 201
+       
+    except Exception, e:
+        return jsonify( {'error' : "could not obtain annotation(s): "+str(e) }), 500
 
-            r = requests.get(url)
-            data = r.json()        
+#List all annotations. build 6
+@api.route('/annotation/all')
```
Comment (FBRTMaka) [inline ooiservices/app/main/annotation.py]: @birdage, I know this is super picky - but maybe just call this route "/annotations" ?  If it's already got it's UI proxy counterpart then just leave it as is as.

File: ooiservices/app/main/annotation.py [hunk 2]
```diff
@@ -23,23 +23,27 @@
...
+    try:
+        url = current_app.config['UFRAME_ANNOTATION_URL'] + current_app.config['UFRAME_ANNOTATION_BASE']+"/find/all"
+        r = requests.get(url)
+        data = r.json() 
+        print data   
 
-            return jsonify( {'annotations' : data['annotations'] }), 201
-        else:
-            return jsonify( {'error' : "no dates specified" }), 500   
+        return jsonify( {'annotations' : data }), 201
```
Comment (FBRTMaka) [inline ooiservices/app/main/annotation.py]: @birdage a "created" status code? (201)

File: ooiservices/app/main/annotation.py [hunk 3]
```diff
@@ -23,23 +23,27 @@
 @api.route('/annotation/<string:instrument>/<string:stream>')
 def get_annotations(instrument,stream):
     try:
-        if 'startdate' in request.args and 'enddate' in request.args:
-            st_date = request.args['startdate']
-            ed_date = request.args['enddate']
+        url = current_app.config['UFRAME_ANNOTATION_URL'] + current_app.config['UFRAME_ANNOTATION_BASE']+"/find/"+instrument
+        r = requests.get(url)
+        data = r.json() 
+        print data
```
Comment (FBRTMaka) [inline ooiservices/app/main/annotation.py]: @birdage print statement snuck in!

File: ooiservices/app/main/annotation.py [hunk 4]
```diff
@@ -23,23 +23,27 @@
...
+        data = r.json() 
+        print data   
 
-            mooring, platform, instrument, stream_type, stream = split_stream_name('_'.join([instrument, stream]))
-            #fixed parameter request            
-            query = '?beginDT=%s&endDT=%s&limit=%s&include_annotations=true&parameters=PD7' % (st_date, ed_date, 3)
-
-            UFRAME_DATA = current_app.config['UFRAME_URL'] + current_app.config['UFRAME_URL_BASE']
-            url = "/".join([UFRAME_DATA,mooring, platform, instrument, stream_type, stream + query])
+        return jsonify( {'annotations' : data }), 201
```
Comment (FBRTMaka) [inline ooiservices/app/main/annotation.py]: @birdage not sure if "created" status code is the droid you're looking for

**Comment by birdage** [comment]
@FBRTMaka @DanielJMaher dan sorry i should of said this really isnt ready yet, i got pulled on to AA, i just wanted to get the code up so i didnt loose any part of it

**Comment by FBRTMaka** [comment]
@birdage, this good to go?

**Comment by DanielJMaher** [comment]
@birdage  Status on this?

**Comment by birdage** [comment]
@DanielJMaher no movement, is this something we want to get in over the next week?

**Comment by birdage** [comment]
@DanielJMaher will update and test

**Pull request merged**